sql/src/main/java/io/crate/metadata/rule/ingest/IngestRulesMetaData.java

```diff
@@ -43,9 +43,25 @@
 
     public static final String TYPE = "ingest_rules";
 
+    /**
+     * Returns a copy of {@link IngestRulesMetaData}
+     */
+    @Nullable
+    public static IngestRulesMetaData copyOf(@Nullable IngestRulesMetaData oldMetaData) {
+        if (oldMetaData == null) {
+            return null;
```

Comment:
this must return a new instance, otherwise you'll get NPE on create/drop. pls add a test for that.